packages/toolkit/src/view/knowledge/components/CatalogFilesTab.tsx

```diff
@@ -0,0 +1,508 @@
+          kbId={knowledgeBase.kbId}
+          accessToken={accessToken}
+          onClose={closeOverlay}
+          showFullFile={true}
+          ownerId={knowledgeBase.ownerName}
+          isOpen={isFileDetailsOpen}
+          setIsOpen={setIsFileDetailsOpen}
+          fileName={selectedFile.name}
+          fileType={selectedFile.type}
+        />
+      )}
+    </div>
+  );
+};
+
+const formatFileSize = (bytes: number | undefined): string => {
+  if (bytes === undefined || isNaN(bytes)) return 'N/A';
+  const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB'];
+  if (bytes === 0) return '0 Bytes';
+  const i = Math.floor(Math.log(bytes) / Math.log(1024));
+  if (i === 0) return bytes + ' ' + sizes[i];
+  return (bytes / Math.pow(1024, i)).toFixed(2) + ' ' + sizes[i];
+};
+
+const formatDate = (dateString: string): string => {
+  const date = new Date(dateString);
+  return date.toLocaleString();
+};
+
+export default CatalogFilesTab;
+
+{/* <div className="flex w-[375px] flex-col gap-3 border-l border-semantic-bg-line pb-8">
+            <div className="flex items-center justify-center gap-3 p-3 pl-3 border-b rounded-tr border-semantic-bg-line bg-semantic-bg-base-bg">
+              <div className="text-semantic-fg-primary product-body-text-3-medium">
+                Preview
+              </div>
+            </div>
+            <div className="flex flex-col h-full gap-3 px-8">
+              <div className="flex flex-col flex-1">
+                <div className="flex flex-col h-full gap-1">
+                  <div className="flex w-[311px] items-center justify-start pb-2.5">
+                    <div className="flex items-start">
+                      <div className="product-body-text-3-semibold">
+                        File name
+                      </div>
+                    </div>
+                  </div>
+                  <Textarea
+                    // {...field}
+                    // id={field.name}
+                    placeholder="Preview"
+                    className="flex-1 flex-grow resize-none"
+                  />
+                </div>
+              </div>
+              <div className="flex items-start justify-end gap-3">
+                <Button
+                  variant="secondaryGrey"
+                  size="lg"
+                  className="capitalize"
+                >
+                  modify
+                </Button>
+                <Button
+                  variant="secondaryGrey"
+                  size="lg"
+                  className="capitalize"
+                >
+                  download
+                  <Icons.DownloadCloud01 className="w-4 h-4 ml-2 stroke-semantic-fg-primary" />
+                </Button>
+              </div>
+              <Button
+                variant="tertiaryDanger"
+                size="lg"
+                className="self-end capitalize"
+              >
+                delete
+              </Button>
+            </div>
+          </div> */}
```

Comment:
@thewbuk Please help us remove this kind of long comment without saying why you want to leave it in the codebase